algorithms/CPlusPlus/Arrays/reverse-array.cpp

```diff
@@ -0,0 +1,29 @@
+#include<iostream>
+using namespace std ;
+int reverse_array(int *arr, int n){
+  
+// Here we will use two pointer approach
+int low = 0 ;     //First pointer at the beginning of the array
+int high = n-1;  //Second pointer at the end of the array
+
+while(low < high){
+  //Swapping array items at the low and high index
+  int temp = arr[low] ;
+  arr[low] = arr[high];
+  arr[high] =temp ;
+  low++;  //Incrementing the low pointer
+  high--; //Decrementing the high pointer
```

Comment:
It seems could be refactor to use only one variable.

Reply:
Yah he can do as he want